extensions/hibernate-envers/runtime/src/main/java/io/quarkus/hibernate/envers/HibernateEnversBuildTimeConfig.java

```diff
@@ -8,14 +10,37 @@
 public class HibernateEnversBuildTimeConfig {
     /**
      * Enable store_data_at_delete feature.
+     * Maps to {@link org.hibernate.envers.configuration.EnversSettings#STORE_DATA_AT_DELETE}.
      */
     @ConfigItem(defaultValue = "false")
     public boolean storeDataAtDelete;
 
     /**
-     * Defines a suffix for historical data table.
+     * Defines a suffix for historical data table. Defaults to {@literal _AUD}.
+     * Maps to {@link org.hibernate.envers.configuration.EnversSettings#AUDIT_TABLE_SUFFIX}.
      */
     @ConfigItem(defaultValue = "_AUD")
-    public String auditTableSuffix;
+    public Optional<String> auditTableSuffix;
+
+    /**
+     * Defines a prefix for historical data table. Default is the empty string.
+     * Maps to {@link org.hibernate.envers.configuration.EnversSettings#AUDIT_TABLE_PREFIX}.
+     */
+    @ConfigItem(defaultValue = "")
+    public Optional<String> auditTablePrefix;
+
+    /**
+     * Revision field name. Defaults to {@literal REV}.
+     * Maps to {@link org.hibernate.envers.configuration.EnversSettings#REVISION_FIELD_NAME}.
+     */
+    @ConfigItem(defaultValue = "REV")
+    public Optional<String> revisionFieldName;
+
+    /**
+     * Revision type field name. Defaults to {@literal REVTYPE}.
+     * Maps to {@link org.hibernate.envers.configuration.EnversSettings#REVISION_TYPE_FIELD_NAME}.
+     */
+    @ConfigItem(defaultValue = "REVTYPE")
+    public Optional<String> revisionTypeFieldName;
```

Comment:
Would you mind adding tests for these? A simple test would be enough, just to check the setting is correctly forwarded to Envers.

I've been bitten by this before: trivial code to forward a setting, but somehow it was not taken into account...

Reply:
done